### DISCUSSION TITLE
Fatal error: Cannot redeclare syslog()

### DISCUSSION THREAD

**Discussion by xuesumin** [FreshRSS/FreshRSS#5176] [Q&A] [answered]
**Describe the bug**
When I upgraded to 1.21.0, the web show the error: 
Fatal error: Cannot redeclare syslog() in /www/wwwroot/rss/lib/lib_rss.php on line 24

**To Reproduce**
1. Go to https://rss.xmfish.org
2. See error:Fatal error: Cannot redeclare syslog() in /www/wwwroot/rss/lib/lib_rss.php on line 24

**Environment information (please complete the following information):**
- Device: vps
- OS: CentOS 7.9
- Browser: chrome
- FreshRSS version: 1.21.0
- Database version: mysql-5.6
- PHP version: php-7.4
- Installation type: Manual install

**Upgrade Steps**
cd /www/wwwroot/rss
wget https://github.com/FreshRSS/FreshRSS/archive/latest.zip
unzip latest.zip
\cp -R -f FreshRSS-latest/* .
chown -R www:www . && chmod -R g+r . && chmod -R g+w ./data/
rm -f latest.zip
rm -f data/do-install.txt
rm -rf FreshRSS-latest/
reboot

**Comment by Alkarex** [comment, marked as answer]
Hello,
This might occur in the case when you have disabled `syslog()` in your PHP settings AND are running PHP < 8.

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/pull/5054

**Comment by xuesumin** [comment]
It works！Thank you！
